### PR TITLE
Feature/cs 29 remove dynamodb.yml dependency

### PR DESCRIPTION
**Pull request by dpetzold**
In order to have a single source of truth the dynamodb configuration has to be pulled from the terraform config. Since terraform does not use non-key attributes support for retrieving column data and overriding the default throughput has been removed.

### CODE REVIEW

**Comment by dpetzold** [comment]
@joconor @brianz this is ready

File: cc_dynamodb/__init__.py [hunk 1]
```diff
@@ -19,27 +18,24 @@
...
-        'yaml': yaml_config,
-        'namespace': namespace
-                        or os.environ.get('CC_DYNAMODB_NAMESPACE'),
-        'aws_access_key_id': aws_access_key_id if aws_access_key_id != False
-                                else os.environ.get('CC_DYNAMODB_ACCESS_KEY_ID', False),
-        'aws_secret_access_key': aws_secret_access_key if aws_secret_access_key != False
-                                    else os.environ.get('CC_DYNAMODB_SECRET_ACCESS_KEY', False),
+        'hcl': terraform_config,
+        'namespace': namespace or os.environ.get('CC_DYNAMODB_NAMESPACE'),
+        'aws_access_key_id': aws_access_key_id or os.environ.get('CC_DYNAMODB_ACCESS_KEY_ID'),
```
Comment (joconor) [inline cc_dynamodb/__init__.py]: No need for default value?

File: cc_dynamodb/__init__.py [hunk 2]
```diff
@@ -19,27 +18,24 @@
...
+        'aws_secret_access_key': aws_secret_access_key or os.environ.get('CC_DYNAMODB_SECRET_ACCESS_KEY'),
```
Comment (joconor) [inline cc_dynamodb/__init__.py]: Same question as above regarding default

**Comment by brianz** [comment]
Outside my area of knowledge but the code looks 👍

**Comment by joconor** [comment]
batman

**Pull request closed without merging**